JL.Core/Mining/Anki/AnkiUtils.cs

```diff
@@ -37,11 +37,21 @@ public static class AnkiUtils
 
     internal static async Task<bool?> CanAddNote(Note note)
     {
-        Response? response = await AnkiConnect.GetCanAddNotesResponse(note).ConfigureAwait(false);
+        Response? response = await AnkiConnect.GetCanAddNotesResponse([note]).ConfigureAwait(false);
         string? resultString = response?.Result?.ToString() ?? null;
 
         return resultString is not null
             ? JsonSerializer.Deserialize<bool[]>(resultString)![0]
             : null;
     }
+
+    internal static async Task<bool[]?> CanAddNotes(Note[] notes)
+    {
+        Response? response = await AnkiConnect.GetCanAddNotesResponse(notes).ConfigureAwait(false);
+        string? resultString = response?.Result?.ToString() ?? null;
+
+        return resultString is not null
+            ? JsonSerializer.Deserialize<bool[]>(resultString)!
```

Comment:
```suggestion
            ? JsonSerializer.Deserialize<List<bool>>(resultString)!
```
Deserializing it to a list instead of an array seems to be faster in this case.